src/cpp/server/server_builder.cc

```diff
@@ -250,6 +250,16 @@ std::unique_ptr<Server> ServerBuilder::BuildAndStart() {
     }
   }
 
+  // == Determine if the server has any callback methods ==
+  bool has_callback_methods = false;
+  for (auto it = services_.begin(); it != services_.end(); ++it) {
+    if ((*it)->service->has_callback_methods()) {
+      has_callback_methods = true;
+      num_frequently_polled_cqs++;
```

Comment:
This isn't really in the scope of this PR, but why do we need to keep num_frequently_polled_cqs as an int when we're treating it as a bool? @vjpai ?

Reply:
Thank you @hcaseyal ! Yes, it can definitely be made a boolean. I kept it as to have a smaller diff, but I'm happy to change it to a `bool`.

Reply:
I think it may make the code easier to read :) Just to sanity check, is it the case that if we have any callback methods, then we have a frequently polled cq?

Reply:
I like @hcaseyal's suggestion of making it a bool since the counting isn't actually important. The callback CQ is indeed frequently polled as are any async API CQs that are passed in and have the frequently-polled bit set.

Reply:
Thank you both. I changed it to @hcaseyal's suggestion.